controllers/resource/template.go

```diff
@@ -253,6 +257,7 @@ func (r *CloudStackTemplate) getEtcdTemplateName(ctx context.Context, eksaCluste
 				return "", err
 			}
 			etcdTemplateName = common.EtcdMachineTemplateName(clusterName, r.now)
+			r.log.Info(fmt.Sprintf("etcd template updated to new name %s", etcdTemplateName))
```

Comment:
```suggestion
			r.log.Info("Etcd template updated", "new name", etcdTemplateName)
```

Same comment for other logs using string interpolation :)